content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,86 @@ ifndef::include-only-once[]
 include::{root-path}_config.adoc[]
 endif::[]
 
-TODO: Add terms and definitions
+3D Asset::
+A 3D model. containing the 3D geometry, combined with assigned material properties and specific meta data.
 
-Static road network::
-Collection of connected roads enriched by static objects that do not change during runtime of a simulation.
+3D Model::
+A (mathematical) geometrical, coordinate-based representation of the surface of an object.
+This excludes physical material properties. For a combination of 3D geometry and physical material properties, see 3D Asset.
+
+3D Environment::
+A 3D representation of the surrounding of a simulated sensor.
+The 3D environment can consist of multiple 3D assets.
+
+Co-Simulation::
+Simulating multiple independent models connected with interfaces.
+A co-simulation platform handles the timing and data transfer between the individual models
+
+Ground truth::
+True static and dynamic states of all objects as defined by content creator or observed by omniscient observer (e.g. simulation platform).
+
+Material::
+A collection of material properties belonging to one entity.
+
+Material assignment::
+A mapping of material IDs to parts of 3D models.
+The material IDs are unique a point to a set of material properties in separate material files.
+
+Material classification::
+Standardized classification scheme that sorts materials into classes and groups of classes, for example, based on their physical properties.
+
+Material Parameter File::
+A file in json format containing defined material properties according to the OpenMATERIAL standard.
+
+Material property::
+Intensive physical or chemical properties of a material 
+
+* Notes: Open Point: What about geometric properties, i.e. roughness, thickness?
+
+Multispectral/hyperspectral::
+A wide range of wavelengths.
+
+Normals (face, vertex normals)::
+A vector that is perpendicular to a given object, surface, vertex or geometrical primitive.
+
+Object::
+An entity to which an object ID can be assigned.
+It comprises discrete meshes and defined 3D boundaries.
+
+Object Part / Subtype::
+(To be defined)
```

Comment:
@ClemensLinnhoff: Not sure whether this term refers to a part/component of an object or a certain type of display feature. Let's discuss in our call this afternoon.

I have the impression that we need to define two concepts:
- Part/component of a 3D object: A signpost consists of two parts, a pole and plate. The pole is represented by a line, the plate is represented by a polygon.
- Material type/feature: Type of geometry used for display. Examples: line, polygon...

Reply:
discuss Geometry later
Subtype will be discussed in vehicle hierarchy

Reply:
replaced with:

Object part::
A segment of an object that has distinct characteristics, for example, material, shape, or function.

Example: A car door differs from the rest of the vehicle due to its function to open and close.